packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_account_account_to_utxos.test.ts

```diff
@@ -0,0 +1,144 @@
+import BigNumber from 'bignumber.js'
+import { JsonRpcClient } from '@defichain/jellyfish-api-jsonrpc'
+import { OP_CODES } from '@defichain/jellyfish-transaction'
+import { AccountToUtxos } from '@defichain/jellyfish-transaction/dist/script/defi/dftx_account'
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { utxosToAccount } from '@defichain/testing'
+import { getProviders, MockProviders } from '../provider.mock'
+import { P2WPKHTransactionBuilder } from '../../src'
+import {
+  findOuts,
+  fundEllipticPair,
+  sendTransaction
+} from '../test.utils'
+import { Bech32, HASH160 } from '@defichain/jellyfish-crypto'
+
+const container = new MasterNodeRegTestContainer()
+let providers: MockProviders
+let builder: P2WPKHTransactionBuilder
+let jsonRpc: JsonRpcClient
+
+beforeAll(async () => {
+  await container.start()
+  await container.waitForReady()
+  await container.waitForWalletCoinbaseMaturity()
+  jsonRpc = new JsonRpcClient(await container.getCachedRpcUrl())
+  providers = await getProviders(container)
+
+  // Prep 1000 DFI UTXOS for testing
+  await container.waitForWalletBalanceGTE(1001)
+})
+
+afterAll(async () => {
+  await container.stop()
+})
+
+beforeEach(async () => {
+  await providers.randomizeEllipticPair()
+  await container.waitForWalletBalanceGTE(11.1)
+  builder = new P2WPKHTransactionBuilder(providers.fee, providers.prevout, providers.elliptic)
+
+  // Fund 10 DFI TOKEN
+  await providers.setupMocks() // required to move utxos
+  await utxosToAccount(container, 10, { address: await providers.getAddress() })
+
+  // Ensure starting balance
+  const account = await jsonRpc.account.getAccount(await providers.getAddress())
+  expect(account).toContain('10.00000000@DFI')
+
+  // Fund 1 DFI utxos for fee
+  await fundEllipticPair(container, providers.ellipticPair, 1)
+})
+
+describe('account.accountToUtxos()', () => {
+  it('should receive token and change', async () => {
+    const destPubKey = await providers.ellipticPair.publicKey()
+    const script = await providers.elliptic.script()
+
+    const conversionAmount = 2.34 // amount converted into utxos
+    const accountToUtxos: AccountToUtxos = {
+      from: script,
+      balances: [{
+        token: 0,
+        amount: new BigNumber(conversionAmount)
+      }],
+      mintingOutputsStart: 2
```

Comment:
The actual `to` address is encoded in the minting outputs?

Reply:
Add assertion cross the the encoded in the DfTx with the listunspent result L92